src/core.h

```diff
@@ -28,6 +28,16 @@
 
 #include "ip_type.h"
 
+#if defined(__APPLE__) || defined(__darwin__)
```

Comment:
can you add ```__OpenBSD__``` to that list as well so we can close #83 as well
please see also the other comments in that issue (for example it's sufficient to check for s6_addr16 only) also about where to put the block